ansible/template/.env.j2

```diff
@@ -0,0 +1,13 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=production
+
+# Change the following values for local development.
+SECRET_KEY = {{ secret_key }}
+API_KEY = {{ trello_api_key }}
+API_TOKEN = {{ trello_api_token }} 
+
+BOARD_ID = "0mFk4pKh"
```

Comment:
Even though these are not very sensitive (e.g. they're fairly useless without the other trello secrets), I would probably still treat them as secrets and hide them